lib/containers/common.pm

```diff
@@ -140,8 +140,11 @@ sub clean_container_host {
         assert_script_run("$runtime rmi --all --force");
     }
     else {
-        assert_script_run("$runtime stop \$($runtime ps -q)", 180) if script_output("$runtime ps -q | wc -l") != '0';
+        my $tmp = "/tmp/podman-ps";
+        assert_script_run "$runtime ps -q | tee $tmp";
+        assert_script_run("$runtime stop \$($runtime ps -q)", 180) if script_run("test -s $tmp") == 0;
```

Comment:
not sure about podman but for docker this is overkill , one line below you also cover this part too

Reply:
check later implementation. the differences between the two container runtime is small on this matter. Anyhow the line was there before this commit.